scipy/_lib/_unuran_utils.py

```diff
@@ -0,0 +1,10 @@
+'''Helper functions to get location of UNU.RAN source files.'''
+
+import pathlib
+from typing import Union
+
+
+def _unuran_dir(ret_path: bool = False) -> Union[pathlib.Path, str]:
+    '''Directory where root unuran/ directory lives.'''
+    p = pathlib.Path(__file__).parent / 'unuran'
+    return p if ret_path else str(p)
```

Comment:
Please only use double quotes to be consistent with the code base.
```suggestion
"""Helper functions to get location of UNU.RAN source files."""

import pathlib
from typing import Union


def _unuran_dir(ret_path: bool = False) -> Union[pathlib.Path, str]:
    """Directory where root unuran/ directory lives."""
    p = pathlib.Path(__file__).parent / "unuran"
    return p if ret_path else str(p)
```